include/RED4ext/CommandLine-inl.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/CommandLine.hpp>
+#endif
+
+#include <RED4ext/Detail/AddressHashes.hpp>
+#include <RED4ext/Relocation.hpp>
+#include <RED4ext/StringView.hpp>
+
+RED4EXT_INLINE RED4ext::CommandLine* RED4ext::CommandLine::Get() noexcept
+{
+    // NOTE: GetAddr is actually necessary here
+
+    static CommandLine* ptr{
+        RED4ext::UniversalRelocPtr<RED4ext::CommandLine>(RED4ext::Detail::AddressHashes::CommandLine).GetAddr()};
+    return ptr;
```

Comment:
I assume this is not a pointer in `data` / `rdata`, but a struct, right? Because if this is a pointer, then this is wrong and it should be:

```suggestion
    static UniversalRelocPtr<CommandLine*> ptr(Detail::AddressHashes::CommandLine);
    return ptr;
```

Reply:
Seems to be a struct, not ptr to struct, yes